CI/file_check.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+FILENAME=$1
+
+DEST="\$(DESTDIR)"
+
+while IFS= read -r LINE_VAL; do
+
+	LINE=$(echo $LINE_VAL | xargs)	
+	IFS=' '
+	read -a COMMAND_ARGUMENTS <<< $LINE
+	LENGTH=${#COMMAND_ARGUMENTS[@]}
+
+	case $LINE in
+		
```

Comment:
Can remove empty line between 'case $LINE in:' and first case, and also between cases.